datalad/support/annexrepo.py

```diff
@@ -426,7 +439,8 @@ def annex_addurls(self, urls, options=[]):
         """
 
         self._run_annex_command('addurl', annex_options=options + urls,
-                                log_online=True, log_stderr=False)
+                                backend=None, log_online=True,
```

Comment:
which all suggests that ideally there should be a unittest ... and `get_file_backend` function?